configure.ac

```diff
@@ -53,6 +53,8 @@ LT_PROG_RC
 AC_PROG_INSTALL
 AC_PROG_LN_S
 
+AM_PATH_PYTHON
+
```

Comment:
This could easily be a separate PR and would make reviewing of the whole easier.